content_scripts/runtime.js

```diff
@@ -19,6 +19,11 @@ var runtime = window.runtime || (function() {
             pageUrlRegex: [],
             clickablePat: /(https?|thunder|magnet):\/\/\S+/ig,
             hintAlign: "center",
+            hintGroups: true,
```

Comment:
Default need to be `false`, since people may have different favor, we also add these settings to document(now only `readme.md`).